client/lib/cart/store/index.js

```diff
@@ -201,12 +201,6 @@ CartStore.dispatchToken = Dispatcher.register( payload => {
 						postalCode = extractStoredCardMetaValue( action, 'card_zip' );
 						countryCode = extractStoredCardMetaValue( action, 'country_code' );
 						break;
-					case 'WPCOM_Billing_MoneyPress_Paygate': {
```

Comment:
I think (not sure), that EBANX cards would still go through that flow. To test, set the currency to BRL, and select Brazil as the country in the payment form.

IIRC we're overwriting the ebanx payment method with "WPCOM_Billing_MoneyPress_Paygate", for historical reasons. It might be that we just need to use "WPCOM_Billing_Ebanx" instead.

Reply:
Forgot about EBANX - for testing, you'll need to fill the Tax ID field/cpf. You can use 527.137.415-72 for that, or any number from https://developers.ebanx.com/guides/going-live/fake-customer-data/fake-brazilian-customer-data/